LibsAndSamples.sln

```diff
@@ -2649,6 +2651,48 @@ Global
 		{C5902FA4-CC11-43E8-B7E3-192B5FFA1C40}.Release|x64.Build.0 = Release|Any CPU
 		{C5902FA4-CC11-43E8-B7E3-192B5FFA1C40}.Release|x86.ActiveCfg = Release|Any CPU
 		{C5902FA4-CC11-43E8-B7E3-192B5FFA1C40}.Release|x86.Build.0 = Release|Any CPU
+		{4BEDCC7F-C027-4326-BAE4-AD62C1CD5572}.Debug + MobileApps|Any CPU.ActiveCfg = Debug + MobileApps|Any CPU

```

Comment:
No files are added, is there a need to change the sln?

Reply:
this project was not part of the solution, so just adding the csproj